data/json/recipes/ammo/rifle.json

```diff
@@ -177,6 +177,47 @@
       [ [ "copper", 3 ] ]
     ]
   },
+  {
+    "result": "reloaded_4570_sp",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_RIFLE",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ "gun", 5 ],
+    "time": 2000,
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 4 ] ],
+    "charges": 1,
+    "using": [ [ "bullet_forming", 15 ], [ "ammo_bullet", 8 ] ],
+    "components": [
+      [ [ "4570_casing", 1 ] ],
+      [ [ "lgrifle_primer", 1 ] ],
+      [ [ "gunpowder", 15 ] ],
```

Comment:
It's literally not possible to get the same energy with black powder.  Black powder burns very differently than smokeless, and you'd either run out of case volume before you got close, or you'd produce a round that blows up your gun, even a modern one.  If they could have produced the same velocities we get today in 1880 with black powder, they would have.

Reply:
That is simply not true. They can, and they did. 
<https://loaddata.com/Cartridge/45-70-Venturino-Black-Powder-Lever-Gun-Data/2695>
These loads are for black powder in trapdoors. They fall very close in line with common commercial smokeless (as above in my table.)
<https://csharpsarms.com/cartridge%20loading%20manual.pdf> (page 22)
These energies look pretty modern to me, 2334 joules? Not a nuclear amount of energy, but still fairly stout.

<https://loaddata.com/Cartridge/45-70-GOEX-Pinnacle-Loads/2508>
<https://archive.org/details/Complete_Guide_to_Handloading_Sharpe_1937/page/n407>
These loads are nearing the overpressure 45-70 loads, some 2678 Joules. and are likely the most you'll get out of blackpowder loads. Yes, it is true you will not be able to stuff enough powder into a case to produce those heavy hitting loads. The most charge I've seen in my searching is about 72grains with the right compression tools.
Sharpe's reloading manual lists information for loading the original 45-70-500 load with black powder, that load produces 2623 Joules.

But in many cases it is entirely possible to get similar energy with black powder, Goex's website lists a wealth of load data, some of which are close to modern energies. See PR #30944 for more information.

>or you'd produce a round that blows up your gun, even a modern one.
Smokeless has a much more intense pressure spike than Blackpowder does. 

From <https://www.starlinebrass.com/articles/Black-Powder-vs-Smokeless-Powder--Comparing-Gunpowder-Types/> :
> When smokeless powder was put into production, it became evident that the designs of guns had to be changed since smokeless powder burns at higher pressures. For example, the French were using a Gras rifle in 11 mm caliber. The only lockup was the bolt handle. While this was okay for black powder, it was not adequate for the newer propellant.

Reply:
I meant in comparison to the loads I already have added.  You aren't moving a 300 grain bullet at 2000FPS out of a standard 18-24" barrel with black powder.  And mind you, if you're reading load data from manuals concerning the Sharps rifle, their test barrels are going to be abnormally long.  I saw 28", 30", even 34" barrels mentioned.  That kind of damage should be considered in the damage multiplier of the Sharps rifle, not the ammo.

I will go ahead and rebalance the current ammo, add a "trapdoor" power loading, and also include a separate recipe for a true black powder loading (if only to make it produce residual smoke like they're talking about in #30944), but be aware that cheapest =/= most common.  My own experience tells me that you're just as, if not more, likely to find Federal or Winchester hunting ammo on a store shelf as you are to find a 405 grain lead cowboy load.

Reply:
Winchester silvertip looks closest to the new 45-70 factory soft-point. <https://www.ableammo.com/catalog/ammo_charts/Winchester_Ammunition_Ballistic.pdf> (data generated from 24" barrel length, sqrt(3192.951J) ~ 56.5

For reloaded 45-70 SP, 90% of the 4570_SP damage = 0.9*57, 51.3. 


The 45-70-500 load blackpowder load listed above in the Sharpe reloading manual (26" barrel) produces 2623J, or 51 damage. The two loads on the load-data website, using more modern black-powder formulations, produce slightly more energy.  That's practically identical, the blackpowder reloads are capable of matching survivor reloaded factory-emulation smokeless loads.

Reply:
I based the new damage for SP off the average of your hunting ammo data, and the damage of the new LFN off your average black powder emulation data, part of which includes the original US Govt .45-70-405 load, and notably lacks a 500 grain load.  From what I'm reading from my article, something replicating the original US Govt .45-70-500 load pushed it's 500 grain bullet at 1210, and that's out of the 32" barrel of a trapdoor Springfield.  That maths out to ~47 damage, again, out of a 32" barrel, and he mentions serious recoil.
https://www.starlinebrass.com/articles/45-70-Government-Loads--45-70-Applications/

Either way, I based the black powder loading off the 405 grain load, not the 500 grain load.  And I used the standard .9 damage multiplier for reloaded ammo as GAME_BALANCE prescribes.  What is it you suggest I change?

Reply:
http://www.frfrogspad.com/springfi.htm

Here's a better reference for classic .45-70-405 and .45-70-500 loads out of the 32" Springfield Rifle and 22" Springfield Carbine.  Again, they only get the numbers you're bringing up that the long 32" barrel can provide.

Reply:
Nothing, then.  I was only suggesting that the middling load could be replicated (in energy, not with the same components)  with black powder. It would, as the sources say, be pushing the limits of the case and cartridge with just black powder.

Reply:
I think it's understood by the devs that handloading in real life can often produce more potent ammunition and not less, but for the sake of game balancing, I think they want factory ammo to be just a little more valuable than ammo you can reload nigh indefinitely with enough skills.

Reply:
Handloading has worse stats in general because of the wide array of materials and conditions it can be performed with, in addition to the very poor quality tools that are available.